### ISSUE TITLE
[Suggestion] Cannot add any list to my Pihole. Connection refused. 

### ISSUE THREAD

**Issue by SamuraiCoder** [lightswitch05/hosts#355]
Hello, first of all thank you for your effort creating this.

I was experiencing troubles to add your lists to my Pihole through the webconfig. I added the lists through "Group Management" > "Ad Lists"

Then I updated gravity to have all configured and this is what I see and therefore they are not added: 

![image](https://user-images.githubusercontent.com/4901895/174354552-84e9defc-50f1-46bd-9b15-e3ffde6d5334.png)

There are not different from other pi-hole lists.  

But if you do, connected to the pi-hole, a `curl` command: 

`curl https://www.github.developerdan.com/hosts/lists/ads-and-tracking-extended.txt`

It responds something about the certificate not being updated. 

The solution is to connect to the pi-hole and do an `apt-get update && apt-get upgrade` and later those adlist can be added again. 

It is something to add to the 'Common issues' section maybe? 

Thanks, Sam.

**Comment by dnmTX** [comment]
> It responds something about the certificate not being updated.

This is `curl` problem not the lists here.
If you're using some Linux flavor i can probably help you navigate that issue.
If you're on Windows and Pi-Hole there is using some kind of a build in `curl`....no idea.

**Comment by SamuraiCoder** [comment]
It's a pi-hole with Debian running Linux. There is no issue because I've fixed it by doing this update. So something done on the pi-hole with this update fixed  it (probably certs) because running again the same command it downloads the list both curl and pi-hole now.

**Comment by dnmTX** [comment]
> with this update fixed it (probably certs)

Yep,that's what it was. Nevertheless,depending on Pi-Hole to update `curl's` certificates it's just not very reliable,to say the least.
There is a easy ways to do it with some simple script and `cron`. Look it up,i'm pretty sure there are bunch of guides out there.

**Comment by lightswitch05** [comment]
Thanks for the info @dnmTX. This list is hosted by GitHub actions, so I have no control over the server anyways.
